src/features/questions/questions-list/question-actions/FavoritePopover.jsx

```diff
@@ -6,9 +6,15 @@ import FavoritePopoverContent from './FavoritePopoverContent';
 export const FavoritePopover = ({ children }) => {
   const { token } = useAuth();
 
+  const text = 'что бы иметь возможность сохранять вопросы';
```

Comment:
заодно и тут опечатку исправить пожалуйста

```suggestion
  const text = 'чтобы иметь возможность сохранять вопросы';
```